app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java

```diff
@@ -425,6 +474,17 @@ public void handleResult(@NonNull final ChannelInfo result) {
             headerSubscribersTextView.setText(R.string.subscribers_count_not_available);
         }
 
+        if (!TextUtils.isEmpty(currentInfo.getSubChannelName())) {
+            headerSubChannelTitleView.setText(String.format(
+                            getString(R.string.channel_created_by),
+                            currentInfo.getSubChannelName())
+            );
+            headerSubChannelTitleView.setVisibility(View.VISIBLE);
+            headerSubChannelAvatarView.setVisibility(View.VISIBLE);
+        } else {
+            headerSubChannelTitleView.setVisibility(View.GONE);
```

Comment:
Also hide headerSubChannelAvatarView

Reply:
I didn't do that, as it's gone by default in the .xml, so I thought that's not needed.